pkg/filters/args.go

```diff
@@ -130,5 +129,29 @@ func (filter *ArgFilter) Parse(filterName string, operatorAndValues string, even
 
 	filter.Filters[id][argName] = val
 
+	filter.Enable()
+
 	return nil
 }
+
+func (filter *ArgFilter) Enable() {
+	filter.enabled = true
+	for _, filterMap := range filter.Filters {
+		for _, f := range filterMap {
+			f.Enable()
+		}
+	}
+}
+
+func (filter *ArgFilter) Disable() {
+	filter.enabled = false
+	for _, filterMap := range filter.Filters {
+		for _, f := range filterMap {
+			f.Disable()
+		}
+	}
+}
```

Comment:
Just a thought for a possible future PR -
I'm not sure how useful it will be to have a generic Enable/Disable methods for this (argument) filter, but letting these methods also accept an event id and argument names, will allow to enable/disable a specific argument filter, which might be useful for changing filters dynamically.

Reply:
We could have a separate function to enable disable an Arg, that's a good idea, but I also think a general Enable Disable is good for feature parity and having a common Filter interface.